Don't install a test database. Who asked for that?
Index: mysql-generic-5.6-5.6.12/share/mysql_system_tables_data.sql
===================================================================
--- mysql-generic-5.6-5.6.12.orig/share/mysql_system_tables_data.sql	2013-06-19 18:20:57.000000000 +0100
+++ mysql-generic-5.6-5.6.12/share/mysql_system_tables_data.sql	2013-07-25 08:31:20.277427914 +0100
@@ -28,15 +28,6 @@
 SELECT REPLACE((SELECT REPLACE(@get_hostname,'_','\_')),'%','\%') INTO @current_hostname;
 
 
--- Fill "db" table with default grants for anyone to
--- access database 'test' and 'test_%' if "db" table didn't exist
-CREATE TEMPORARY TABLE tmp_db LIKE db;
-INSERT INTO tmp_db VALUES ('%','test','','Y','Y','Y','Y','Y','Y','N','Y','Y','Y','Y','Y','Y','Y','Y','N','N','Y','Y');
-INSERT INTO tmp_db VALUES ('%','test\_%','','Y','Y','Y','Y','Y','Y','N','Y','Y','Y','Y','Y','Y','Y','Y','N','N','Y','Y');
-INSERT INTO db SELECT * FROM tmp_db WHERE @had_db_table=0;
-DROP TABLE tmp_db;
-
-
 -- Fill "user" table with default users allowing root access
 -- from local machine if "user" table didn't exist before
 CREATE TEMPORARY TABLE tmp_user LIKE user;
